app/src/main/java/org/wikipedia/bridge/CommunicationBridge.java

```diff
@@ -49,7 +49,7 @@ public CommunicationBridge(final WebView webView) {
         webView.getSettings().setAllowUniversalAccessFromFileURLs(true);
         webView.getSettings().setMediaPlaybackRequiresUserGesture(false);
         webView.setWebChromeClient(new CommunicatingChrome());
-        webView.addJavascriptInterface(new BridgeMarshaller(), "marshaller");
```

Comment:
Now the in-article link does not work as expected.

Received `Uncaught ReferenceError: marshaller is not defined`.

Reply:
Good catch! fixed.